src/shutdownscreen.cpp

```diff
@@ -76,8 +76,6 @@ void ShutdownScreen::applySystemState(DeviceState::DeviceState::StateIndication
 {
     switch (what) {
         case DeviceState::DeviceState::Shutdown:
-            // To avoid early quitting on shutdown
-            HomeApplication::instance()->restoreSignalHandlers();
```

Comment:
Can we remove the whole method now from HomeApplication?

Reply:
Possibly. Depends on lifecycle of signal handling code & data. On quick glance I'd assume it is not a problem and cleanup code could be removed. But at least in theory there might be some reason to leave the code in and even call it (is this part of lipstick a library or plugin? can it be unloaded on exit path? do we want to pay special attention to signal received after leaving mainloop before exit? etc)

Reply:
To me it looked liked dead code if this is removed.

Reply:
As the PR stands - yes, it is dead code. I'll add a separate cleanup commit - easy to revert if real need ever arises. (Well, even then it probably would be better to reset default handlers rather than restore previous handlers that might well reside in address space of some already unloaded plugin or stte).